backend/src/bundles/recently-viewed/recently-viewed.repository.ts

```diff
@@ -97,6 +99,28 @@ class RecentlyViewedRepository implements IRecentlyViewedRepository {
 
         return deletedItem ? true : false;
     }
+
+    public async findRecentlyViewedResumesWithCount(): Promise<
+        RecentlyViewedResumesWithCount[]
+    > {
+        const interval = '1 day';
```

Comment:
What do you think about making `interval` a parameter of the `findRecentlyViewedResumesWithCount` method and setting its default value to '1 day'?
`public async findRecentlyViewedResumesWithCount(interval: string = '1 day')`.

This will allow you to call `findRecentlyViewedResumesWithCount` with different intervals without changing the method.